spec/patterns/java_spec.rb

```diff
@@ -16,3 +16,31 @@
     end
   end
 end
+
+describe "JAVASTACKTRACEPART" do
+  let(:pattern) { 'JAVASTACKTRACEPART' }
+  let(:message) { '  at com.sample.stacktrace.StackTraceExample.aMethod(StackTraceExample.java:42)' }
+  it "matches" do
+    grok = grok_match(pattern, message, true)
+    expect(grok).to include({
+                                "message"=>"  at com.sample.stacktrace.StackTraceExample.aMethod(StackTraceExample.java:42)",
+                                "method"=>"aMethod",
+                                "class"=>"com.sample.stacktrace.StackTraceExample",
+                                "file"=>"StackTraceExample.java",
+                                "line"=>"42"
+                            })
+  end
+
+  context 'generated file' do
+    let(:message) { '  at org.jruby.RubyMethod$INVOKER$i$call.call(RubyMethod$INVOKER$i$call.gen)' }
+    it "matches" do
+      grok = grok_match(pattern, message, true)
+      # TODO does not match
```

Comment:
:yawning_face:

Reply:
> 🥱

Better: `- [ ]` (with a space between the braces) gives you an actual checkbox in Github ;-)

Reply:
thanks, unfortunately I am still reluctant in "fixing" (changing) pattern definitions here, in context of ECS.
initially, thought this to be addressed "later" - maybe I will start cleaning up the base-line in parallel...

Reply:
moved to https://github.com/logstash-plugins/logstash-patterns-core/pull/272